website/index.html

```diff
@@ -112,6 +112,17 @@
         width: 1.2em;
         vertical-align: text-bottom;
       }
+
+    @media (prefers-color-scheme: dark) {
+      a:link {color:#268BD2;}
+      a:visited {color:#6C71C4;}
+      body {background-color:  #002B36; color: white;}
+      td {color:black;}
+    }
+    @media (prefers-color-scheme: light) {
+      body {background:#fff; color: #333; }
+    }
```

Comment:
Could we move most of this into main.css?

Reply:
I moved most of it into main  darkening table color's didn't seem to work when they were in main though.